fiduswriter/languagetool/static/js/modules/lt/editor.js

```diff
@@ -49,7 +49,7 @@ export class EditorLT {
                 type: 'menu',
                 disabled: _editor => !this.supportedLanguages.includes(
                     this.editor.view.state.doc.firstChild.attrs.language
-                ) || this.editor.docInfo.access_rights !== 'write',
+                ) || this.editor.docInfo.access_rights !== 'write' || !this.editor.ws.isOnline(),
```

Comment:
We should probably change to this:
```js
editor.docInfo.access_rights !== 'write' || !editor.ws.isOnline()
```
And then change the `_editor` to `editor`. The udnerscroe is just used to mark that the `editor` is not in use. But now it actually is in use.